modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -31,41 +31,63 @@ endif::[]
 
 . Stop all of the machine configuration pools managed by the Machine Config Operator (MCO):
 
-** Stop the master configuration pool:
+** Stop the `master` configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool master --type='merge' --patch \
   '{ "spec": { "paused": true } }'
 ----
 
-** Stop the worker machine configuration pool:
+** Stop the `worker` machine configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused": true } }'
 ----
 
 . To prepare for the migration, set the migration field to `null` by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
```

Comment:
Before this step, users shall run `oc get Network.config cluster -o jsonpath='{.status.migration}'` to ensure the `.status.migration` field is empty in the network.config object.

Reply:
Thanks. Added:

```
. Check that the migration status is empty for the `Network.config.openshift.io` object by entering the following command. Empty command output indicates that the object is not in a migration operation.
+
[source,terminal,subs="+quotes"]
----
$ oc get Network.config cluster -o jsonpath='{.status.migration}'
----
```

If the object is undergoing migration, what should a user do?

Reply:
This command `$ oc patch Network.operator.openshift.io cluster --type='merge' \
  --patch '{ "spec": { "migration": null } }'` tells CNO to clear `.status.migration` from the network.config object. So users shall wait until it becomes empty, if it is not.